lib/emailnotifications_queue_handler.ts

```diff
@@ -198,12 +200,12 @@ async function handleNotification(
   const eventContent = {
     addressSource: emailNotification.addressSource,
     messageId,
-    mta: "sendgrid",
-    notificationId
+    notificationId,
+    transport: "sendgrid"
```

Comment:
can we made the transport configurable ?

Reply:
you mean the actual transport used by nodemailer or the value of the attribute in the event?